hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -10,9 +10,9 @@ tests:
   - interval: 1m
     input_series:
       - series: 'node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate{namespace="ci",pod="virt-controller-8546c99968-x9jgg",node="node1"}'
-        values: "2 2 2 2 2 2 2 2 2 2"
+        values: '2+0x10'
```

Comment:
@Barakmor1 I know this maybe out of scope, but can you please add a comment that refers to this [link](https://github.com/prometheus/prometheus/blob/83a56e22ab2c51e79e125dad5d56107d3b0dd2fc/docs/configuration/unit_testing_rules.md#test-file-format) which explains the test file format?

Reply:
Done.